tests/test_validator_integration.py

```
@@ -0,0 +1,967 @@
+        tree = ast.parse(source)
+        forwarded = self._extract_train_model_kwargs(tree)
+        assert "shuffle_types" not in forwarded
+
+    def test_random_drop_removed_from_schema(self):
+        """training.random_drop is no longer in schema (dead config removed)."""
+        source = self._get_launch_training_source()
+        tree = ast.parse(source)
+        forwarded = self._extract_train_model_kwargs(tree)
+        assert "random_drop" not in forwarded
+
+    def test_run_name_forwarded(self):
```

Ruff (0.15.1)

[error] Redefinition of unused `test_run_name_forwarded` from line 323: `test_run_name_forwarded` redefined here

(F811)